content/funding-your-training/_bursaries-and-scholarships-cta.html.erb

```diff
@@ -1,6 +1,6 @@
 <%= render CallsToAction::SimpleComponent.new(
   icon: "icon-arrow",
-  text: "Find out how much is awarded for training to teach certain subjects through bursaries and scholarships.",
+  text: "Find out more information about bursaries and scholarships.",
```

Comment:
Do you 'find out information' or 'find information'?

Reply:
Trying to get across that it's more than we've shown in this table... 

Find further information?